src/components/actionSheet/index.js

```diff
@@ -59,6 +59,27 @@ export default class ActionSheet extends BaseComponent {
      * When passed (only with useNativeIOS), will display a cancel button at the bottom (overrides cancelButtonIndex)
      */
     showCancelButton: PropTypes.bool,
+    /**
+     * Add or override style of the action sheet
+     */
+    sheetStyle: ViewPropTypes.style,
+    /**
+     * Add or override style of the dialog wrapping the action sheet
+     */
+    dialogStyle: ViewPropTypes.style,
+    /**
+     * Add or override style of the options list
+     */
+    listStyle: ViewPropTypes.style,
+    /**
+     * Render custom title
+     */
+    renderTitle: PropTypes.func,
+    /**
+     * Render custom action<br>
```

Comment:
<br>???

Reply:
Not sure what is the problem here, is it the naming? - It is the same convention as in Picker

Reply:
The tag `br`?

Reply:
Done